image_provisioner/playbooks/roles/cri-o/tasks/main.yml

```diff
@@ -0,0 +1,98 @@
+---
+- block:
+    - name: Ensure container-selinux is installed
+      package:
+        name: container-selinux
+        state: present
+
+    # Used to pull and install the system container
+    - name: Ensure atomic is installed
+      package:
+        name: atomic
+        state: present
+
+    - name: Ensure runc is installed
```

Comment:
Do you have three different package commands because that's what they did upstream?

Reply:
Yes, we need atomic package to be installed to pull the crio image and install it.

Reply:
Yes, of course we do. I mean do we literally need discrete tasks to install each rather than a loop `with_items`?

Reply:
You are right, we don't need discrete tasks. Updated the PR with the changes.